EXILED/Exiled.API/Features/Respawn.cs

```diff
@@ -283,11 +272,12 @@ public static int GetTickets(SpawnableFaction faction)
         /// <summary>
         /// Forces a spawn of the given <see cref="SpawnableTeamType"/>.
         /// </summary>
-        /// <param name="team">The <see cref="SpawnableTeamType"/> to spawn.</param>
-        public static void ForceWave(Faction team)
+        /// <param name="faction">The <see cref="SpawnableTeamType"/> to spawn.</param>
+        /// <param name="mini">Docs.</param>
+        public static void ForceWave(Faction faction, bool mini = false)
         {
-            if (TryGetWaveBase(team, out SpawnableWaveBase wave))
-                ForceWave(wave);
+            SpawnableWaveBase waveBase = WaveManager.Waves.Find(x => x.TargetFaction == faction && (mini && x is IMiniWave));
```

Comment:
```suggestion
            SpawnableWaveBase waveBase = WaveManager.Waves.Find(x => x.TargetFaction == faction && (mini == x is IMiniWave));
```

Reply:
I hope I understand that part correctly.

Reply:
what I had was correct, `mini` is the parameter that determines whether the user wants a minu wave or a normal wave, all 4 waves (2 mini waves and 2 normal ones) are stored in `WaveManager.Waves`, the `x` here is an instance of a wave